index.js

```diff
@@ -164,7 +172,7 @@ module.exports = function shim(file) {
   function write(buf) { content += buf; }
   function end() {
     var messages = [];
-    resolveShims(file, messages, function (err, info) {
+    resolveShims(file, messages, shimConfig$file, function (err, info) {
```

Comment:
Please stick to variable names used in remaining code, i.e. `shimConfigFile`, no `$`.